_includes/sidebar.html

```diff
@@ -9,13 +9,61 @@
         <span class="pt-icon-standard pt-icon-feed sidebar-icon"></span>
         <a href="/">News</a>
       </div>
+      <div
```

Comment:
You can add news about these additional pages in `index.html`, something like:

```
<div class="pt-card card__title">
    <h3 class="card__title__text">APIO 2020 registration, schedule, rules, and onsite competition</h3>
    <p class="card__title__right">6 February 2020</p>
  </div>
  <div class="pt-card card__content">
    <p>
      We have added several pages in the website, including registration, schedule, rules, and onsite competition. Please do check them.
    </p>
  </div>
```

Put the link to the page for each of the `registration`, `schedule`, `rules`, and `onsite competition` in the post.